Assets/UnityPharusAPI/Player/ATrackingEntity.cs

```diff
@@ -76,7 +77,7 @@ public float Speed
         /// <summary>
         /// A list of the track's echoes (feet) as Vector2
```

Comment:
Please change the type from "list" to "ConcurrentDictionary" in this summary as well so IntelliSense won't mislead us in the future.

Reply:
Done in [57ca92f](https://github.com/Playful-Interactive-Environments/UnityTrackingClient/pull/1/commits/57ca92f1aac0620059a8137dc976bd9214fd3ee8)